### ISSUE TITLE
Official package, version 1.0.3 does not have any Image.this[] or Image.Pixel - 100% using the new version

### ISSUE THREAD

**Issue by speciesunknown** [SixLabors/ImageSharp#1757]
### Prerequisites

- [ *] I have written a descriptive issue title
- [* ] I have verified that I am running the latest version of ImageSharp
- [* ] I have verified if the problem exist in both `DEBUG` and `RELEASE` mode
- [ *] I have searched [open](https://github.com/SixLabors/ImageSharp/issues) and [closed](https://github.com/SixLabors/ImageSharp/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
I have seen this issue before but the reason was due to another provider having an outdated version in nuget.

I have verified that this is not the case. I have the latest version, 1.0.3.

This code which appears in the documentation does not compile as there is no Pixel or this[] or other visible method to access individual pixels.

![image](https://user-images.githubusercontent.com/602738/131560436-6a67d154-5a88-4dde-b7f3-ce4bb888a112.png)

### System Configuration
.net core 2.1

- ImageSharp version: 1.0.3
- Other ImageSharp packages and versions:
- Environment (Operating system, version and so on):
- .NET Framework version:
- Additional information:
Author(s): Six Labors and contributors
Date published: Wednesday, February 17, 2021 (2/17/2021)

**Comment by antonfirsov** [comment]
Exactly, this change happened long time ago [in beta 7](https://sixlabors.com/posts/beta-7/).

If performance matters, use `.GetPixelRowSpan(y)` instead of the `[x, y]` indexer: https://docs.sixlabors.com/articles/imagesharp/pixelbuffers.html#efficient-pixel-manipulation

However be aware that the above method will be replaced in ImageSharp 2.0 unfortunately, see #1739.
